test/components/CalendarDay_spec.jsx

```diff
@@ -75,6 +76,23 @@ describe('CalendarDay', () => {
         expect(wrapper.prop('aria-label')).to.equal('chooseAvailableDate text');
       });
 
+      it('is formatted with the dateIsSelected phrase function when day is selected', () => {
+        const selectedModifiers = new Set(['selected', 'selected-start', 'selected-end']);
+
+        selectedModifiers.forEach((selectedModifier) => {
+          const modifiers = new Set().add(selectedModifier);
+
+          const wrapper = shallow(<CalendarDay
```

Comment:
Not a huge deal, but the `(` should end a line here, and the `)` should begin once - the rest of the indentation will follow that. (You may need to double-wrap the jsx in parens to satisfy the linter)

Reply:
Same as above comment, this is the style that is used throughout this entire file so I was just going with the flow, but I can certainly change it up.

Reply:
the other one is fine, but this one goes against (the spirit, at least, of) the airbnb styleguide, so we should clean this pattern up.

Reply:
I'll do a quick sweep and clean them up.